.github/actions/certify-openshift-images/entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -eou pipefail
 
 docker login -u mongodb+mongodb_atlas_kubernetes -p "${QUAY_PASSWORD}" quay.io
 
-DIGESTS=$(docker manifest inspect "quay.io/${REPOSITORY}:${VERSION}" | jq -r .manifests[].digest)
+DIGESTS=$(docker manifest inspect "quay.io/${REPOSITORY}:${VERSION}" | jq -r '.manifests[] | select(.platform.os!="unknown") | .digest')
```

Comment:
we are not supposed to have unknown images/digests. Should we avoid this happening instead?

Reply:
@helderjs We thought its because of `COPY` commands. cc: @s-urbaniak

Reply:
yes, we have 3 layers here:
1. architecture-dependent: file system hierarchy and binaries
2. architecture-dependent: our operator binary
3. architecture-independent: trusted CA bundle
4. architecture-independent: license files

Reply:
xref https://github.com/mongodb/mongodb-atlas-kubernetes/blob/ed2e4772b445bb790d0a7f5bda948249ca1dab03/fast.Dockerfile#L24-L26